package.json

```diff
@@ -66,7 +66,7 @@
   },
   "peerDependencies": {
     "@edx/frontend-platform": "^7.0.0 || ^8.0.0",
```

Comment:
>[!IMPORTANT]
> Update  frontend-platform to a version compatible with `paragon 23`

Reply:
Moved this message to the PR description, thanks!